pkg/cmd/server/start/start_node.go

```diff
@@ -303,6 +306,19 @@ func StartNode(nodeConfig configapi.NodeConfig, components *utilflags.ComponentF
 		config.EnsureDocker(docker.NewHelper())
 	}
 
+	// If config specifies a perFSGroup quota to apply to empty dir volumes, we
```

Comment:
This needs to be encapsulated in a helper, it has no business being embedded here.  Also, you should only be initializing this when the kubelet is enabled.

Create a new config.EnsureLocalQuota() and put it in the block above.